CustomerContactBook/Controllers/GroupMembersController.cs

```diff
@@ -35,40 +36,40 @@ public async Task<ActionResult<List<GroupMemberModel>>> GetGroupMembers()
             return result;
         }
 
-        // GET: api/GroupMembers/5
+
         /// <summary>
         /// get groupmember with same id
         /// </summary>
         /// <param name="id">id of group member</param>
-        [HttpGet("{Cid}/{Gid}")]
+        [HttpGet("{CustomerId:long}/{GroupId:long}")]
         [ProducesResponseType(typeof(GroupMemberModel), StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
-        public async Task<ActionResult<GroupMemberModel>> GetGroupMember(long Cid, long Gid)
+        public async Task<ActionResult<GroupMemberModel>> GetGroupMember(long CustomerId, long GroupId)
```

Comment:
please use camel casing. You may find more about naming conventions on the [learn.microsoft.com/...](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#:~:text=When%20writing%20method%20parameters%2C%20use%20camel%20casing)

Reply:
Fixed this in the latest commit